src/dotnet/commands/dotnet-compile-native/ILCompilerInvoker.cs

```diff
@@ -43,32 +37,29 @@ private void InitializeArgs(NativeCompileSettings config)
             var coreLibsPath = Path.Combine(config.IlcSdkPath, "sdk");
             foreach (var reference in Directory.EnumerateFiles(coreLibsPath, "*.dll"))
             {
-                argsList.Add($"-r");
-                argsList.Add($"{reference}");
+                argsList.Add($"-r:{reference}");
```

Comment:
Want to point out this will cause this arg to be quoted and escaped as a single arg. 

So for a space with a path, it will end up like this: `"-r:/some path/with/a/space"

Is this intentional?